config/config.ts

```diff
@@ -271,27 +271,67 @@ const config: Configuration = {
-                            '* DX11 recommended \n\n' +
-                            '* HIGH or lower texture resolution setting recommended \n\n',
+                            description: 'Includes our 8K full resolution cabin, cockpit and exterior textures. This is the full fidelity ' +
+                                'experience and our recommendation if your system is powerful enough to support it. Realistic and in high ' +
+                                'detail.\n\n' +
+                                '* DX11 recommended \n\n' +
+                                '* HIGH or lower texture resolution setting recommended \n\n',
+                            isExperimental: false,
+                            releaseModel: {
+                                type: 'fragmenter',
+                            },
+                        },
+                        {
+                            name: 'Development (4K)',
+                            key: 'a380x-dev-4k',
+                            url: 'https://flybywirecdn.com/addons/a380x/master-4k',
+                            alternativeUrls: [ ],
+                            description: 'Development will have the latest features that will end up in the next stable. ' +
+                                'Although every change is QA-tested bugs are a little more likely. It updates whenever something is added to ' +
+                                'the \'master\' branch on Github. Please visit our discord for support. \n\n' +
+                                'Includes our 4K downscaled cabin, cockpit and exterior textures. Choose this option for reduced ' +
+                                'stutters, better performance, with HIGH or lower texture resolution. Especially, if you intend to use the ' +
+                                'following:\n\n' +
+                                '* Use frame generation \n\n' +
+                                '* Virtual Reality (VR) \n\n' +
+                                '* DX12 beta \n\n' +
+                                '* or are otherwise limited by your graphics card VRAM amount. ',
+                            isExperimental: false,
+                            releaseModel: {
+                                type: 'fragmenter',
+                            },
+                        },
+                        {
+                            name: 'Development (8K)',
+                            key: 'a380x-dev-8k',
+                            url: 'https://flybywirecdn.com/addons/a380x/master-8k',
+                            alternativeUrls: [ ],
+                            description: 'Development will have the latest features that will end up in the next stable. ' +
+                                'Although every change is QA-tested bugs are a little more likely. It updates whenever something is added to ' +
```

Comment:
```suggestion
                                'Although every change is QA-tested, bugs are a little more likely. It updates whenever something is added to ' +
```
Add comma for better reading flow